Sezai/index.js

```diff
@@ -0,0 +1,27 @@
+// 1.
+let str1 = "Hello World";
+console.log("str1 ==> ", str1.toUpperCase());
+
+// 2.
+let str2 = "Hello Earthling";
+console.log("str2 ==> ", str2.toLowerCase());
+
+// 3.
+console.log(str1.length);
+let str3 = "Bananas"
+console.log("str3 ==> ", str1.slice(0,1)+str3+str1.slice(0,1));
```

Comment:
The logic is correct but you need the first character of the given string which is str3
console.log("str3 ==> ", str3.slice(0,1)+str3+str3.slice(0,1));